README.md

```diff
@@ -1,3 +1,37 @@
-# Jellyfin Chromecast
+<h1 align="center">Jellyfin Chromecast Client</h1>
```

Comment:
Jellyfin Chromecast Web Receiver or Jellyfin Chromecast Receiver App might be better names?

Reply:
I agree `Jellyfin Chromecast Receiver (app)` would be better suited. But not sure if there is a different name that refers to this repo. Setting it to your suggestion now but i'm open to any suggestions.